packages/clerk-js/src/ui/components/UserProfile/MfaPhoneCodeScreen.tsx

```diff
@@ -39,34 +42,21 @@ export const MfaPhoneCodeScreen = withCardStateProvider(() => {
         title={localizationKeys('userProfile.mfaPhoneCodePage.title')}
         resourceRef={ref}
       />
-      <SuccessPage
```

Comment:
We should keep the `<SuccessPage/>` here as we need to show to the user the backup codes.